cli/display.go

```diff
@@ -3,8 +3,12 @@ package cli
 import (
 	"context"
 	"fmt"
+	"os"
+	"strings"
 	"time"
 
+	surterm "gopkg.in/AlecAivazis/survey.v1/terminal"
```

Comment:
from github or gopkg.in you have to choose only one

Reply:
done